packages/foreman/rubygem-multi_json/rubygem-multi_json.spec

```diff
@@ -1,72 +1,76 @@
-%{!?scl:%global pkg_name %{name}}
```

Comment:
We can actually already drop this package it appears. We only use this from the Ruby on Rails SCL and not from this repo.

Reply:
Query  - This package  `rubygem-multi_json` listed under key `foreman_scl_and_nonscl_packages` in package_manifest.yaml. So, you mean this should be under `foreman_scl_packages` only?

Reply:
I didn't get this. could you please elaborate on this? 
I found this multi_json package under `foreman_scl_and_nonscl_packages` in package_manifest.yaml.

Reply:
It's supposed to be only under `foreman_nonscl_packages` since the RoR SCL already provides us one for the SCL context. It's fine to leave it like this for now.

Reply:
Do you want me to revert this as well like jwt?